jarbas/api/views.py

```diff
@@ -30,6 +30,11 @@ def get(self, request):
         if self._bool_param('suspicions'):
             self.queryset = self.queryset.suspicions()
 
+        # filter has_receipt_url
+        has_receipt_url = self._bool_param('has_receipt_url')
```

Comment:
You documented `has_receipt`, but here you used `has_receipt_url`… can you fixed that, please? I think we can change it here (and have `receipt_url` instead of `has_receipt_url` as the variable name).

If write a test for this view with the filter would be great too ; )

Reply:
I wrote filters for has_receipt and has_no_receipt, commented inline, if it's what you meant. :)